src/components/transaction_history/transaction_history.ts

```diff
@@ -0,0 +1,84 @@
+import { html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { DateTime } from "luxon";
+import { transactionHistoryCSS } from "./transaction_history.css";
+import { centsToDollars } from "../../utils";
+
+const headerLabels = ["Date", "Description", "Amount"];
+
+interface TransactionHistoryItem {
+  "created_at": string,
+  "line_item_status": "VALID" |  "INVALID" | "OFFSET" | "PENDING" | "AUTHORIZED" | "DECLINED" | "VOID" | "POSTED"
+  "line_item_type": "CHARGE" |  "PAYMENT" | "CREDIT_OFFSET" | "DEBIT_OFFSET" | "MANUAL_FEE" | "REFUND" | ""
```

Comment:
Why is empty string an option for `line_item_type`?

Reply:
Idk. I branched off the amount story branch. I’ll revisit tomorrow and see. Thanks for pointing it out